wiki-webui/src/main/java/org/exoplatform/wiki/webui/control/action/SavePageActionComponent.java

```diff
@@ -142,6 +142,7 @@ protected void processEvent(Event<SavePageActionComponent> event) throws Excepti
         markup = markup.trim();
   
         String newPageName = TitleResolver.getId(title, false);
+        pageParams = new WikiPageParams(pageParams.getType(), pageParams.getOwner(), newPageName);

```

Comment:
Why creating a new object WikiPageParams instead of calling pageParams.setPageName(newPageName) ?